apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -252,7 +259,32 @@ router.post(
           await createAndLinkLineitem(instance, job, assessment_metadata);
         }
       });
-      return res.redirect(`/pl/jobSequence/${serverJob.jobSequenceId}`);
+      return res.redirect(res.locals.urlPrefix + '/jobSequence/' + serverJob.jobSequenceId);
```

Comment:
Since the page has edit components, and I can't think of why you'd need to use it as a Student Data Viewer, I think limiting it to just Student Data Editors is reasonable.

Did I implement that right in 864ee0983?

Reply:
Yeah, that looks good to me!